utils/event-proxy-server/payload-files/express/test-error--transaction.json

```diff
@@ -118,28 +219,19 @@
         "message": "Example app listening on port 3030"
       }
     ],
-    "request": {
```

Comment:
The `request` is not included in transactions anymore with v8 (the error events still have it). In transactions, the url is included in `contexts.trace.data.url`.